app/celer-sim/RunnerInputIO.json.cc

```diff
@@ -100,7 +100,7 @@ void from_json(nlohmann::json const& j, RunnerInput& v)
     LDIO_LOAD_REQUIRED(initializer_capacity);
     LDIO_LOAD_REQUIRED(secondary_stack_factor);
     LDIO_LOAD_REQUIRED(use_device);
-    LDIO_LOAD_OPTION(sync);
+    LDIO_LOAD_OPTION(action_times);
```

Comment:
Can you add next to the other `LDIO_LOAD_DEPRECATED`:

```c++
    LDIO_LOAD_DEPRECATED(sync, action_times);
```

Reply:
should we do the same for celer-g4?

Reply:
It couldn't hurt. I don't think we have as many inputs for it though: we *do* want `celer-sim` to work across a couple different versions so that we can run our peformance regression tests.